pkgs/os-specific/bsd/openbsd/pkgs/libc/libpthread-no-werror.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/lib/librthread/Makefile b/lib/librthread/Makefile
+index 67a2678574d..6bc8d407d76 100644
+--- a/lib/librthread/Makefile
++++ b/lib/librthread/Makefile
+@@ -3,7 +3,7 @@
+ LIB=pthread
+ LIBCSRCDIR=    ${.CURDIR}/../libc
+
+-CFLAGS+=-Wall -g -Werror -Wshadow
++CFLAGS+=-Wall -g -Wshadow
```

Comment:
Could we use NIX_CFLAGS_COMPILE rather than a patch?

Reply:
With `-Wno-error`?

Reply:
Yeah.

Reply:
done